adhoc_modules/data/cron_data.xml

```diff
@@ -7,6 +7,7 @@
            <field name="interval_type">days</field>
            <field name="numbercall">-1</field>
            <field name="model_id" ref="model_db_configuration"/>
+           <field name="state">code</field>
```

Comment:
mmmm esto es necesario? este campo es nuevo y es necesario en todos los cron?

Reply:
@jjscarafia yes, i think it is, this is the reason why https://youtu.be/EaOzmo5fhhI

Reply:
great @zaoral , thanks for the video explanation